imblearn/utils/tests/test_validation.py

```diff
@@ -46,17 +46,16 @@ def test_check_ratio_error():
 
 
 def test_ratio_all_over_sampling():
+    _expected_ratio = {1: 50, 2: 0, 3: 75}
```

Comment:
Not convince since we don't do anything with check_ractio neither with the expected values.